subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/java/JavaLibraryPlugin.java

```diff
@@ -46,16 +50,37 @@ public void execute(CopySpec copy) {
             }
         });
 
+
+        final File dependenciesFile = new File(project.getBuildDir(), "shipkit-dependencies");
```

Comment:
Let's add extension to the file for clarity. I suggest different name, too. "runtime-dependencies", "dependency-information", "dependency-info". I like "dependency-info" most. When you change the name, let's update the task name so that it is consistent with new naming. Consistency is king :)

Reply:
What kind of extension did you have in mind? At the moment file doesn't have a common format, just one dependency per line with dependency properties delimeted by ":". It would be easy to make it "csv", a little more effort would be required to make it "json". Do you think it's worth? Maybe it would be more readable for debugging purposes

Reply:
OK JSON will be simple and much more readable, so in my opinion it would be the best choice.